src/backend/events/builtin/twitch-event-source.js

```diff
@@ -21,7 +21,36 @@ module.exports = {
                 icon: "fad fa-siren-on",
                 getMessage: (eventData) => {
                     const showUserIdName = eventData.username.toLowerCase() !== eventData.userDisplayName.toLowerCase();
-                    return `**${eventData.userDisplayName}${showUserIdName ? ` (${eventData.username})` : ""}** raided with **${eventData.viewerCount}** viewer(s)`;
+                    return `**${eventData.userDisplayName}${
+                        showUserIdName ? ` (${eventData.username})` : ""
+                    }** raided with **${eventData.viewerCount}** viewer(s)`;
+                }
+            }
+        },
+        {
+            id: "raid-sent-off",
+            name: "Raid Sent Off",
+            description: "When your raid finishes sending off",
```

Comment:
End the description with a period to match existing convention:
![simulate raid](https://github.com/user-attachments/assets/b5308b96-8a3e-44e8-9d9a-5e88a8665ef1)